<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWMilHojas</class>
 <widget class="QMainWindow" name="MWMilHojas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MWMilHojas</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lImagen">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>831</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Carpeta destino</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lECarpetaDestino">
            <property name="text">
             <string>D:\MIL_HOJAS\resultado</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBDirDestino">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>371</width>
          <height>327</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>DPIS Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEDPISResultadoFolios">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Img.Frente</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEImagenFrenteHojas">
             <property name="text">
              <string>D:\MIL_HOJAS\FRENTE.JPG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBFrenteHojas">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Img.Izq</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEImagenIzquierdaHojas">
             <property name="text">
              <string>D:\MIL_HOJAS\IZQUIERDA.JPG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBIzquierdaHojas">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Img.Dch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEImagenDerechaHojas">
             <property name="text">
              <string>D:\MIL_HOJAS\DERECHA.JPG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBDerechaHojas">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Img.Trasera</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEImagenTraseraHojas">
             <property name="text">
              <string>D:\MIL_HOJAS\TRASERA.JPG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBTraseraHojas">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Alto Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEAltoResultadoFolios">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Ancho Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEAnchoResultadoFolios">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Largo Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lELargoResultadoFolios">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pBGenerateHojas">
           <property name="text">
            <string>Generar Folios</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="frame_7">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>411</width>
          <height>261</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Imagen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEImagenTiras">
             <property name="text">
              <string>IMG-20151010-WA0000.jpg</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBImagenParaTiras">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>DPIS Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEDPISResultadoTiras">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Ancho Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEAnchoResultadoTiras">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_25">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Alto Resultado</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEAltoResultadoTiras">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_27">
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Corte</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lECorteTiras">
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="cBDobles">
           <property name="text">
            <string>Dobles</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBGenerate">
           <property name="text">
            <string>Generar Tiras</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
